calico_containers/calicoctl.py

```diff
@@ -467,6 +477,25 @@ def checksystem(fix=False, quit_if_error=False):
             print >> sys.stderr, "WARNING: ipv6 forwarding is not enabled."
             system_ok = False
 
+    # Check docker comparability
+    try:
+        info = docker_client.version()
+    except docker.errors.APIError:
+        print >> sys.stderr, "ERROR: Docker server must support Docker Remote API v1.16 or greater."
+        system_ok = False
+    else:
+        api_version = normalize_version(info['ApiVersion'])
+        # Check that API Version is at least 1.16
+        if cmp(api_version, normalize_version("1.16")) < 0:
+            print >> sys.stderr, "ERROR: Docker server must support Docker " \
+                                 "Remote API v1.16 or greater."
+            system_ok = False
+
+        # Check that if Remote API Version is 1.20, they are using the experimental docker build
```

Comment:
No. As discussed remove the 1.20 block when checking into master. Then when we merge master into libnetwork-rebased we'll update this code.